examples/email-ner/README.md

```diff
@@ -1,10 +1,10 @@
 # Email-NER
 
-Email Named Entity Recognition is an expample to extract entities in an email. It is worked out by TIP Spyre squad. Official repository is [here](https://github.ibm.com/redstack-power/spyre)
+Email Named Entity Recognition is an expample to extract entities in an email. It is worked out by TIP Spyre squad. Official repository is [here](https://github.ibm.com/redstack-power/spyre). It is using LLM inferencing to extract the Named Entities in the given text. So need to use [vLLM](../vllm/) provided by PIM as base image.
 
 ### Build
+Ensure to replace the `FROM` image in [Containerfile](Containerfile) with the vLLM image you have built before building this image.
```

Comment:
Is it an optional step to replace vLLM image with custom one?

Reply:
Not an optional, user needs to change it, since the current image used is in private registry right. But once we open sourced the images that can be used here as is.

Reply:
Okay